keras/backend/mxnet_backend.py

```diff
@@ -953,8 +989,10 @@ def _normalize_axis(axis, ndim):
         for i, a in enumerate(axis):
             if a is not None and a < 0:
                 axis[i] = a % ndim
+    elif axis is None:
```

Comment:
Is this correct? mxnet use (,) to indicate all reduce. It seems keras use (,) to indicate do nothing